Games/types/Mafia/items/Gun.js

```diff
@@ -10,15 +10,21 @@ module.exports = class Gun extends Item {
         this.mafiaImmune = options?.mafiaImmune;
         this.cursed = options?.cursed;
 
+        this.baseMeetingName = "Shoot Gun";
+        this.currentMeetingIndex = 0;
+
         this.meetings = {
-            "Shoot Gun": {
+            [this.baseMeetingName]: {
                 actionName: "Shoot",
                 states: ["Day"],
                 flags: ["voting", "instant", "noVeg"],
                 action: {
                     labels: ["kill", "gun"],
                     item: this,
                     run: function () {
+                        this.item.drop();

```

Comment:
had to move this up before the dominates check, else the gunslinger picks up the gun and this function will drop it